kasa/smart/smartdevice.py

```diff
@@ -149,6 +148,21 @@ async def _negotiate(self):
         if "child_device" in self._components and not self.children:
             await self._initialize_children()
 
+    def request_renegotiation(self) -> None:
```

Comment:
I bet this is not the nicest way to handle it, but this will clear the state to force the next `update()` perform renegotiation. Improvement ideas are welcome, we need to revisit this later to support pairing new devices without HA restart.